manifest.json

```
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "product-list",
-  "version": "0.29.0",
+  "version": "0.29.1",
```

No need to update this

```suggestion
  "version": "0.29.0",
```